packages/wallet/src/views/Home.jsx

```diff
@@ -0,0 +1,190 @@
+  }, [address])
+
+  return (
+    <Page>
+      <div className='relative'>
+        <div className='hidden lg:block lg:fixed lg:-top-24 lg:-left-52 lg:z-0'>
+          <img alt='kernel fingerprint' src={linesVector} width={383} height={412} />
+        </div>
+        <div className='hidden lg:block lg:fixed lg:-top-12 lg:-right-52 lg:z-0'>
+          <img alt='kernel fingerprint' src={linesVector} width={442} height={476} />
+        </div>
+      </div>
+      <div className='mt-32 mx-8 sm:mx-24 xl:mx-48'>
+        <div className='text-center'>
+          <div className='font-heading lg:text-5xl text-5xl text-primary lg:py-5'>
+            Welcome, {nickname}.
+          </div>
+          <hr />
+          <div className='text-2xl my-4 text-secondary'>
+            You are already valuable, just as you are.
+          </div>
+          <div className='my-4 text-secondary'>
+            This portal will help you learn how to realize some of that value in the
+            world of web3. We're so happy you're with us.
+          </div>
+        </div>
+
+        <div className='my-4 px-4 grid grid-cols-1 md:grid-cols-2 md:my-16 md:px-16'>
+          <div>
+            <h3 className='font-heading text-center text-3xl text-primary py-5'>For Everyone</h3>
+            <div className='grid grid-cols-1 md:grid-cols-2 md:gap-x-8 gap-y-8 border-0 md:border-r border-kernel-grey md:pr-12'>
+              {everyoneCards.map((everyoneCard, index) => {
+                return (
+                  <a key={index} href={`${everyoneCard.active ? everyoneCard.url : '#'}`}>
+                    <div className={
+                      `${everyoneCard.active ? 'bg-kernel-dark text-kernel-white' : 'bg-kernel-grey'} p-5 rounded shadow-md`
+                    }
+                    >
+                      <div className='text-xl mb-2'>{everyoneCard.title}</div>
+                      <div className='text-base mb-1'>{everyoneCard.description}</div>
+                    </div>
+                  </a>
+                )
+              })}
+            </div>
+          </div>
+          <div>
+            <h3 className='font-heading text-center text-3xl text-primary py-5'>Kernel Additions</h3>
+            <div className='grid grid-cols-1 gap-y-8'>
+              {kernelCards.map((kernelCard, index) => {
+                return (
+                  <a key={index} href={`${kernelCard.active ? kernelCard.url : '#'}`} className='w-full md:w-72 my-0 mx-auto'>
+                    <div className={
+                      `${kernelCard.active ? 'bg-kernel-dark text-kernel-white' : 'bg-kernel-grey'} p-5 rounded shadow-md`
+                    }
+                    >
+                      <div className='text-xl mb-2'>{kernelCard.title}</div>
+                      <div className='text-base mb-1'>{kernelCard.description}</div>
+                    </div>
+                  </a>
+                )
+              })}
```

Comment:
Since this is now being duplicated, you can break out this returned component into a named component defined at the top level, like
```
const LinkCard = (card, index) => {
  ...
}
```